conf/bridge-server.conf

```diff
@@ -1,5 +1,5 @@
-synapse.user = your-username-here
-synapse.api.key = your-api-key-here
+synapse.user = Bridge.Study.Creator
+synapse.api.key = SW6jpgyt8ot4Hv2yAyIbEn5WTWi9jSOgbL/DPaM3/eEo//P8Cmy+w/dfN6NCblpRs0R2yWdwTzHR4qyHy1dsvg==
```

Comment:
You'll have to remove this from the PR, and then regenerate this key, de-activating this in Synapse. This shouldn't be checked into source control, it should be handled like other keys